app/views/shared/fields/_show_raw.erb

```diff
@@ -5,66 +8,71 @@
 
 <% if collectionsView %>
   <% notesToExclude << 'description' %>
-  <%= render :partial => 'shared/fields/note', :locals => {:type => 'description'} %>
+  <%= render :partial => 'shared/fields/note', :locals => {:type => 'description'}.merge(component) %>
 <% end %>
-<%= render :partial => 'shared/fields/collection' %>
-<%= render :partial => 'shared/fields/relationships' %>
-<%= render :partial => 'shared/fields/note', :locals => {:type => 'performers'} %>
-<%= render :partial => 'shared/fields/date' %>
-<%= render :partial => 'shared/fields/note', :locals => {:type => 'extent'} %>
-<%= render :partial => 'shared/fields/note', :locals => {:type => 'abstract'} %>
-<%= render :partial => 'shared/fields/scope_of_content' %>
-<%= render :partial => 'shared/fields/cartographic' %>
-<%= render :partial => 'shared/fields/note', :locals => {:type => 'table of contents'} %>
+<%= render :partial => 'shared/fields/collection', :locals => component %>
+<%= render :partial => 'shared/fields/relationships', :locals => component %>
+<%= render :partial => 'shared/fields/note', :locals => {:type => 'performers'}.merge(component) %>
+<%= render :partial => 'shared/fields/date', :locals => component %>
+<%= render :partial => 'shared/fields/note', :locals => {:type => 'extent'}.merge(component) %>
+<%= render :partial => 'shared/fields/note', :locals => {:type => 'abstract'}.merge(component) %>
+<%= render :partial => 'shared/fields/scope_of_content', :locals => component %>
+<%= render :partial => 'shared/fields/cartographic', :locals => component %>
+<%= render :partial => 'shared/fields/note', :locals => {:type => 'table of contents'}.merge(component) %>
 <%= render :partial => 'shared/fields/note', :locals => {:exclude_type => notesToExclude,
-        :exclude_label => ['digital object made available by'] } %>
-<%= render :partial => 'shared/fields/resource_type' %>
-<%= render :partial => 'shared/fields/language' %>
+        :exclude_label => ['digital object made available by'] }.merge(component) %>
+<% unless defined?(componentIndex) %>
```

Comment:
for consistency, can the resource_type partial also use the new Hash#merge approach rather than `unless defined?(componentIndex)`?